bin/critterget.py

```diff
@@ -62,16 +63,42 @@
 MONEY_VALUE = 'moneyValue'
 MEAN_DURATION = 'meanDuration'
 
-BASEURL = "https://developers.crittercism.com/v1.0/"
+SUMMARY_ATTRIBUTES = [
+    'appName',
+    'appType',
+    'crashPercent',
+    'dau',
+    'latency',
+    'latestAppStoreReleaseDate',
+    'latestVersionString',
+    'linkToAppStore',
+    'iconURL',
+    'mau',
+    'rating',
+    'role',
+    'appVersions'
+]
+
+CRASH_ATTRIBUTES = [
+    'hash',
+    'lastOccurred',
+    'sessionCount',
+    'uniqueSessionCount',
+    'reason',
+    'status',
+    'displayReason',
+    'name'
+]
+
+BASEURL = "https://developers.crittercism.com/v2/"
+
 
 def apicall_with_response_code(uri, attribs=None):
     # perform an API call
-    if (debug): print u'access token is {}'.format(access_token)
     url = BASEURL + uri
 
-    if (attribs): url += "?"+attribs
-
-    if (debug): print u'reqstring is {}'.format(url)
+    if debug:
```

Comment:
Moving debug outside of global makes this and other functions break unless main() is called first.  Why not have a global debug var?